domenetjenester/behandling-prosessering/src/main/java/no/nav/ung/sak/behandling/prosessering/task/OppfriskInntektForBehandlingerPåVentBatchTask.java

```diff
@@ -15,12 +15,12 @@
 import java.util.Map;
 
 @ApplicationScoped
-@ProsessTask(value = OppfriskInntektForBehandlingerPåVentBatchTask.TASKTYPE, cronExpression = "0 0 12 * * *")
+@ProsessTask(value = OppfriskInntektForBehandlingerPåVentBatchTask.TASKTYPE, cronExpression = "0 * * * *")
```

Comment:
For meg så ser det ikkje ut til at det støttes å droppe sekunder her, sjølv dette er optional i standarden for cron. ProsessTaskHandlerRef initierer alltid med withSeconds satt til true og då vil det feile fordi den kun finner 5 deler.